client-web/src/functional-e2e/space/pages/SpacePage.ts

```
@@ -0,0 +1,98 @@
+  }
+
+  async navigateToSettings() {
+    await this.settingsTab.click();
+  }
+
+  async verifySpaceTitle(title: string) {
+    await expect(this.page.getByRole('heading', { name: title })).toBeVisible();
+  }
+
+  async verifyTutorialsPresent(minCount: number = 2) {
+    const tutorialHeadingPattern = /^[👋📚↪️🤝⚙️🧩🧹]/;
```

Biome (2.1.2)

[error] A character class cannot match a surrogate pair. Add the 'u' unicode flag to match against them.

A surrogate pair forms a single codepoint, but is encoded as a pair of two characters. Without the unicode flag, the regex matches a single surrogate character.
Safe fix: Add unicode u flag to regex


(lint/suspicious/noMisleadingCharacterClass)